modules/react-native-zoom-free-calling/store/api.js

```diff
@@ -0,0 +1,52 @@
+import axios from "axios";
+import options from "../options";
+import base64 from "react-native-base64";
+
+const BASE_URL = "https://api.zoom.us";
```

Comment:
Can we make this a module option?

Reply:
This is the base URL for Zoom APIs, this will be the same for everyone.